todo_app/data/trello_items.py

```diff
@@ -0,0 +1,42 @@
+from flask import Flask, render_template, request
+
+import requests
+import os
+
+def get_trello_cards():
+    url = f'https://api.trello.com/1/boards/{os.getenv("TRELLO_BOARD_ID")}/lists'
+
+    print(os.getenv("API_KEY"))
```

Comment:
For Trello your API KEY isn't particularly sensitive (that's really the token) but it still gives me a jump to see it logged to the console outside of a dev environment - we should probably remove this!